src/ServicePulse.Host/vue/src/components/failedmessages/MessageView.vue

```diff
@@ -71,51 +76,64 @@ async function getConfiguration() {
 }
 
 async function getEditAndRetryConfig() {
-  const response = await useFetchFromServiceControl("edit/config");
-  const data = await response.json();
-  configuration.value.edit = data;
+  const [, data] = await useTypedFetchFromServiceControl<EditAndRetryConfig>("edit/config");
+
+  editAndRetryConfiguration.value = data;
   return;
```

Comment:
not required

Reply:
not required?

Reply:
i don't know what i was referring to anymore...